src/flint/routers/api.py

```diff
@@ -55,11 +61,41 @@ async def chat(
         user = await sync_to_async(user.get)()
     uuid = user.khojuser.uuid
 
+    # Set user message to the body of the request
+    user_message = Body
+    # Check if message is an audio message
+    if MediaUrl0 is not None and MediaContentType0 is not None and MediaContentType0.startswith("audio/"):
+        audio_url = MediaUrl0
```

Comment:
We should log what media types people are sending! Options so far:
- text
- audio

But can include things like img, pdf in the future.